stack/indexer/rpm/builder.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -x
```

Comment:
Debug element?

Reply:
Additional information generator, I think it's required.